tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -12,6 +12,14 @@ namespace Datadog.Trace
     /// </summary>
     public static partial class Tags
     {
+        /// <summary>
+        /// The name of the instrumented service. Its value is usually constant for the lifetime of a process,
+        /// but can technically change for each trace if the user sets it manually.
+        /// This tag is added during MessagePack serialization using the value from <see cref="SpanContext.ServiceName"/>
+        /// or <see cref="Tracer.DefaultServiceName"/>.
+        /// </summary>
+        public const string Service = "service";
```

Comment:
I don't think we should expose this publicly personally (plus you need to update the public test settings etc if you do)

Reply:
Fixed in 67766c252ff33326bf698ee218663977ea104548 so it is internal